crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -175,14 +175,23 @@ def openstack_check
         ret = {}
         # swift replicas check vs. number of disks
         prop = Proposal.where(barclamp: "swift").first
-        return ret if prop.nil?
-        replicas = prop["attributes"]["swift"]["replicas"] || 0
+        unless prop.nil?
+          replicas = prop["attributes"]["swift"]["replicas"] || 0
```

Comment:
Well, it was not needed until this PR. Do you really think there should be separate PR for that?